app/resolve/pyproject.toml

```diff
@@ -29,6 +29,7 @@ trino={version = "^0.328.0", extras = ["sqlalchemy"]}
 python-dotenv = "1.0.1"
 mysqlclient="^2.1.1"
 pyodbc="^4.0.35"
+pytest="8.3.3"
```

Comment:
Please add this to a `test` group - 
https://python-poetry.org/docs/managing-dependencies/#dependency-groups